### PR TITLE
Credentials now come from the awssdk

### PR DESCRIPTION
**Pull request by marcbowes**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -34,7 +32,7 @@ itertools = "0.10.0"
 thiserror = "1.0.29"
 anyhow = "1.0.44"
 tokio = "1.12.0"
-serde = "1.0.130"
+serde = { version = "1.0.130", features = ["derive"] }
```
Comment (marcbowes) [inline Cargo.toml:35]: This feature was being enabled via rusoto.

File: src/awssdk_driver.rs [hunk 2]
```diff
@@ -135,17 +134,23 @@ The following error may have more information: {}
...
+                        .profile_name(name)
+                        .build(),
+                )
+                .build(),
+        ),
+        None => aws_config,
+    };
+    let aws_config = aws_config.load().await;
 
+    let conf = config::Builder::from(&aws_config).region(env.current_region());
```
Comment (marcbowes) [inline src/awssdk_driver.rs:153]: This is a little confusing (the way regions work), and I think will require another PR to cleanup. For now, the behavior is the same as before (which may/not be buggy, but definitely is hard to understand).

**Comment by brack** [comment]
credential error comes back much faster. thank you!!

**Comment by brack** [comment]
can resolve this upon merge: https://github.com/awslabs/amazon-qldb-shell/issues/182

File: src/awssdk_driver.rs (hunk 2 above)
Comment (rcoh) [reply src/awssdk_driver.rs:153]: you should be able to set the region on `aws_config` above. You may want to use `RegionProviderChain::first_try(env.current_region()).or_else_default()`

Comment (marcbowes) [reply src/awssdk_driver.rs:153]: Agreed with the direction, but there is some subtlety in the shell which lets people changes regions on the fly. I need to fix some of that madness first, which is sort of what I was alluding to.

Comment (marcbowes) [reply src/awssdk_driver.rs:153]: I'll link the next PR to this one so it's clear.

**Review by guyilin-amazon — APPROVED** [review]

**Pull request merged**